src/store/AppSlice.js

```diff
@@ -1,3 +1,6 @@
+const isOpfsEnabled = (process.env.OPFS_IS_ENABLED || 'false').toLowerCase() === 'true'
```

Comment:
I think you need to merge?  There's a new condition here based on our convo yesterday https://github.com/bldrs-ai/Share/blob/main/src/store/AppSlice.js

Reply:
PTAL